src/main/java/pivotaltracker/ui/pages/workspace/WorkspaceTrackerPage.java

```diff
@@ -0,0 +1,32 @@
+package pivotaltracker.ui.pages.workspace;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import pivotaltracker.BasePage;
+
+/**
+ * WorkspaceTrackerPage class
+ *
+ * @author Limbert Vargas
+ * @version 1.0
+ */
+public class WorkspaceTrackerPage extends BasePage {
+    @FindBy(id = "tracker")
+    private WebElement workspaceTracker;
```

Comment:
what is workspace traker??

Reply:
is the page that shows when I create a new workspace page